src/widgets/Breakdown/components/Breakdown.js

```diff
@@ -18,24 +23,28 @@ const updatedTime = updated => {
   return null;
 };
 
-const Breakdown = ({ heading, data, description }) =>
-  <div className="panel Breakdown">
-    <div className="panel-heading">
-      {heading}
-    </div>
-    <div className="panel-body hidden-xs">
-      <div className="row">
-        {data.results && <VerticalBarChart data={getKeyValuePairs(data.results)} />}
+const Breakdown = ({ connectDragSource, connectDragPreview, connectDropTarget, isDragging, isOver, ...props }) =>
+  compose(connectDragSource, connectDropTarget)(
+    <div className="panel Breakdown" style={{ opacity: isDragging || isOver ? 0.5 : 1 }}>
```

Comment:
`opacity: isDragging || isOver ? 0.5 : 1`
This is the same everywhere and should be factored out to its own method/file. The values can be put in the constants.